app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailsActivity.java

```diff
@@ -50,20 +54,24 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_event_details);
 
-        ButterKnife.bind(this);
+        Event initialEvent = getIntent().getParcelableExtra(EVENT_KEY);
+
+        if(initialEvent == null) {
```

Comment:
> The case will be when programmer forgets to pass the event to the activity. We can't trust anyone.

So this is for development stage right?

>Showing a blank screen is worse than crashing the app.

At developing stage crashing the app will be better option as it will suggest something wrong. No need to adjust that case.